spyder/plugins/debugger/widgets/framesbrowser.py

```diff
@@ -149,13 +155,33 @@ def setup(self):
         self.finder.sig_hide_finder_requested.connect(
             self.sig_hide_finder_requested)
 
+        # Widget empty pane
+        self.pane_empty = PaneEmptyWidget(
+            self,
+            "debugger",
+            _("Not debugging right now"),
+            _("Start a debugging session with the ⏯ button, allowing you to "
+              "step through your code and see the functions that "
+              "Python has run.")
```

Comment:
```suggestion
            _("Debugging is not active"),
            _("Start a debugging session with the ⏯ button, allowing you to "
              "step through your code and see the functions here that "
              "Python has run.")
```

@conradolandia, I think you applied an older version of the suggestion we were discussing about this afternoon. Please apply this one now, which has the latest changes.